klog.go

```diff
@@ -1265,6 +1265,54 @@ func Infof(format string, args ...interface{}) {
 	logging.printf(infoLog, logging.logr, format, args...)
 }
 
+// InfoS structured logs to the INFO log.
+func InfoS(msg string, keysAndValues ...interface{}) {
+	b := &bytes.Buffer{}
+	kvMap, _ := kvList2Map(keysAndValues)
+	if kvMap == nil {
+		return
+	}
+
+	b.WriteString(msg)
+
+	for key, val := range kvMap {
+		appendKeyValue(b, key, val)
+	}
+
+	logging.println(infoLog, logging.logr, b)
+}
+
+func appendKeyValue(b *bytes.Buffer, key string, value interface{}) {
+	if b.Len() > 0 {
+		b.WriteByte(' ')
+	}
+	b.WriteString(key)
+	b.WriteByte('=')
+
+	stringVal, ok := value.(string)
```

Comment:
Could we just use `fmt.Sprint` for strings ? I don't think there is any significant performance improvement by adding this code.

Reply:
done